docs/DeveloperGuide.md

```diff
@@ -157,47 +152,47 @@ This section describes some noteworthy details on how certain features are imple
 
 #### Proposed Implementation
 
-The proposed undo/redo mechanism is facilitated by `VersionedAddressBook`. It extends `AddressBook` with undo/redo
-history, stored internally as an `addressBookStateList` and `currentStatePointer`. Additionally, it implements the
+The proposed undo/redo mechanism is facilitated by `VersionedStoreMando`. It extends `StoreMando` with undo/redo
+history, stored internally as an `storeMandoStateList` and `currentStatePointer`. Additionally, it implements the
 following operations:
 
-* `VersionedAddressBook#commit()` — Saves the current location book state in its history.
-* `VersionedAddressBook#undo()` — Restores the previous location book state from its history.
-* `VersionedAddressBook#redo()` — Restores a previously undone location book state from its history.
+* `VersionedStoreMando#commit()` — Saves the current location book state in its history.
+* `VersionedStoreMando#undo()` — Restores the previous location book state from its history.
+* `VersionedStoreMando#redo()` — Restores a previously undone location book state from its history.
 
-These operations are exposed in the `Model` interface as `Model#commitAddressBook()`, `Model#undoAddressBook()`
-and `Model#redoAddressBook()` respectively.
+These operations are exposed in the `Model` interface as `Model#commitStoreMando()`, `Model#undoStoreMando()`
+and `Model#redoStoreMando()` respectively.
 
 Given below is an example usage scenario and how undo/redo mechanism behaves at each step.
 
-Step 1. The user launches the application for the first time. The `VersionedAddressBook` will be initialized with the
+Step 1. The user launches the application for the first time. The `VersionedStoreMando` will be initialized with the
 initial location book state, and the `currentStatePointer` pointing to that single location book state.
 
 ![UndoRedoState0](images/UndoRedoState0.png)
 
 Step 2. The user executes `delete 5` command to delete the 5th item in the address book. The `delete` command
-calls `Model#commitAddressBook()`, causing the modified state of the address book after the `delete 5` command executes
-to be saved in the `addressBookStateList`, and the `currentStatePointer` is shifted to the newly inserted address book
+calls `Model#commitStoreMando()`, causing the modified state of the address book after the `delete 5` command executes
+to be saved in the `storeMandoStateList`, and the `currentStatePointer` is shifted to the newly inserted address book
 state.
 
 ![UndoRedoState1](images/UndoRedoState1.png)
 
-Step 3. The user executes `add n/David …​` to add a new item. The `add` command also calls `Model#commitAddressBook()`,
-causing another modified address book state to be saved into the `addressBookStateList`.
+Step 3. The user executes `add n/David …​` to add a new item. The `add` command also calls `Model#commitStoreMando()`,
+causing another modified address book state to be saved into the `storeMandoStateList`.
```

Comment:
As mentioned above, this could be "causing another modified inventory state to be saved into the `storeMandoStateList`."

Reply:
Thanks!